aws/resource_aws_eip.go

```diff
@@ -441,7 +447,8 @@ func resourceAwsEipDelete(d *schema.ResourceData, meta interface{}) error {
 	case ec2.DomainTypeVpc:
 		log.Printf("[DEBUG] EIP release (destroy) address allocation: %v", d.Id())
 		input = &ec2.ReleaseAddressInput{
-			AllocationId: aws.String(d.Id()),
+			AllocationId:       aws.String(d.Id()),
+			NetworkBorderGroup: aws.String(d.Get("network_border_group").(string)),
```

Comment:
This is required else delete keeps retrying with an `InvalidAddressID.NotFound` error code when the Network Border Group specifies a Wavelength (or Local) zone.

Reply:
Nit: Rather than always sending a potentially empty string value, its generally safer to `d.GetOk` instead. The AWS APIs do not seem to mind since the information is always included but it could cause issues with AWS "compatible" APIs that do not implement network border groups and error when they do not understand the request.

Reply:
True, I hadn't thought of that case. `network_border_group` is always set during Read for the "real" AWS API.